terraform/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "vpc" {
+  description = "VPC object copied from variable to the next stage"
+  value       = juju_application.opensearch.placement
+}
```

Comment:
**todo:** re. [the spec](https://docs.google.com/document/d/1EG71A2pJ244PQRaGVzGj7Mx2B_bzE4U_OSqx4eeVI1E/edit?tab=t.0#heading=h.sothcshk3dl1), I would suggest adding `app_name` and `requires`/`provides` outputs. I believe you mentioned this in the README.